spec/system/provider_interface/candidate_pool/provider_views_candidate_pool_list_spec.rb

```diff
@@ -216,4 +248,72 @@ def and_i_expect_to_see_the_total_results_count
   def and_i_expect_to_see_the_updated_results_count
     expect(page).to have_content('1 candidate found')
   end
+
+  def first_subject
+    @first_subject ||= Subject.select("name, string_agg(id::text, ',') as id").group(:name).order(:name).first
+  end
+
+  def last_subject
+    @last_subject ||= Subject.select("name, string_agg(id::text, ',') as id").group(:name).order(:name).last
+  end
+
+  def when_i_add_all_remaining_filters
+    check("subject-#{first_subject.id}-field")
+    check("subject-#{last_subject.id}-field")
+    check('study-mode-full-time-field')
+    check('study-mode-part-time-field')
+    check('course-type-tda-field')
+    check('course-type-he-hes-sd-ss-sc-ssc-ta-field')
+    check('visa-sponsorship-not-required-field')
```

Comment:
Not a blocker, but could we use the actual text for these? It makes it easier to replicate the tests manually when something breaks.